lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1700,7 +1700,7 @@ if (!sourceList)
 
         if((loadScriptFlag & LoadScriptFlag_isByteCodeBufferForLibrary) == LoadScriptFlag_isByteCodeBufferForLibrary)
         {
-            grfscr |= (fscrNoAsmJs | fscrNoPreJit);
```

Comment:
grfscr gets encoded into library bytecode as part of function body. We don't set it in JsHost, so we don't need to set it in ch either (checked with @MikeHolman)